.fixtures.yml

```diff
@@ -2,5 +2,10 @@
 # See https://github.com/puppetlabs/puppetlabs_spec_helper#using-fixtures for details
 ---
 fixtures:
-  forge_modules:
-#     stdlib: "puppetlabs/stdlib"
+  repositories:
+    stdlib: https://github.com/puppetlabs/puppetlabs-stdlib.git
+    apache: https://github.com/puppetlabs/puppetlabs-apache.git
+    htpasswd: https://github.com/citrininfo/puppet-htpasswd.git
+    vcsrepo: https://github.com/puppetlabs/puppetlabs-vcsrepo.git
+    git: https://github.com/puppetlabs-toy-chest/puppetlabs-git.git
+    concat: https://github.com/puppetlabs/puppetlabs-concat.git
```

Comment:
@DLeich might it be better to refer to the Forge versions, instead of the bleeding edge versions on GitHub?

Reply:
no :) we refer to master/main branch for unit tests, so wen can detect breaking changes in dependencies before they are released. for acceptance tests we use the latest versions, that the metadata.json accepts, from the forge (this module currently has no acceptance tests but I will add them later

Reply:
Right on! Makes sense.

I've got a module I've been struggling to add acceptance tests to; would love to see how you do that here or elsewhere.